src/server/index.ts

```diff
@@ -53,16 +89,20 @@ const createAndApplyActions = (): void => {
 
   app.post<unknown, DripResponse, BotRequestType>('/bot-endpoint', (req, res) => {
     const { address, amount, sender } = req.body;
+    metrics.data.total_requests++;
```

Comment:
Maybe ok, but looks a bit imperatively

Reply:
The general architecture does not make it super convenient to report metrics. Here we get a metric that could help and it does not cost much.